lib/sharedPart.js

```diff
@@ -146,8 +146,8 @@ class SharedPart {
     }
   }
 
-  static #readMainLiquid(name, templateConfig) {
-    const mainPartPath = `./${this.TEMPLATE_FOLDER}/${name}/${templateConfig.text}`;
```

Comment:
@Woetfin I think the issue is your `config.json` file, the `text` property.
In my opinion, we should use that property here rather calling the `{name}.liquid`